mdoc/Test/DocTest-framework-inheritance.cs

```diff
@@ -18,6 +18,9 @@ public class MyClass
     #if FXTWO
         : MyBaseClassTwo
     #endif
+    #if FXTHREE
```

Comment:
this could have been simplified by simply adding `|| FXTHREE` to the first conditional preprocessor directive ... but it's not a huge deal :)